scripts/globals/items/kitron_macaron.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------------
+-- ID: 5781
+-- Item: kitron_macaron
+-- Food Effect: 30Min, All Races
+-----------------------------------------
+-- Increases rate of synthesis success +7%
+-- Increases synthesis skill gain rate +7% 
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0;
+    if (target:hasStatusEffect(EFFECT_FOOD) == true or target:hasStatusEffect(EFFECT_FIELD_SUPPORT_FOOD) == true) then
```

Comment:
(food uses the item id to link it up with the item script, which fov food doesn't have)